Features/ManageabilityPkg/Library/ManageabilityTransportSsifLib/Dxe/ManageabilityTransportSsif.c

```diff
@@ -0,0 +1,387 @@
+                                           AcquireTransportSession function.
+  @param [out]  TransportAdditionalStatus  The additional status of specific transport
+                                           interface after the reset.
+                                           NULL means no additional status of this
+                                           transport interface.
+
+  @retval      EFI_SUCCESS              Transport interface status is returned.
+  @retval      EFI_INVALID_PARAMETER    The invalid transport token.
+  @retval      EFI_TIMEOUT              The reset process is time out.
+  @retval      EFI_DEVICE_ERROR         The transport interface has problems to return
+                                        status.
+               Otherwise                Other errors.
+
+**/
+EFI_STATUS
+EFIAPI
+SsifTransportReset (
+  IN  MANAGEABILITY_TRANSPORT_TOKEN              *TransportToken,
+  OUT MANAGEABILITY_TRANSPORT_ADDITIONAL_STATUS  *TransportAdditionalStatus OPTIONAL
+  )
+{
+  return EFI_UNSUPPORTED;
+}
+
+/**
+  This function transmit the request over target transport interface.
+  The generic EFI_STATUS is returned to caller directly after reseting transport
+  interface. The additional information of transport interface could be optionally
+  returned in TransportAdditionalStatus to describes the status that can't be
+  described obviously through EFI_STATUS.
+  See the definition of MANAGEABILITY_TRANSPORT_ADDITIONAL_STATUS.
+
+  @param [in]  TransportToken           The transport token acquired through
+                                        AcquireTransportSession function.
+  @param [in]  TransferToken            The transfer token, see the definition of
+                                        MANAGEABILITY_TRANSFER_TOKEN.
+
+  @retval      The EFI status is returned in MANAGEABILITY_TRANSFER_TOKEN.
+
+**/
+VOID
+EFIAPI
+SsifTransportTransmitReceive (
+  IN  MANAGEABILITY_TRANSPORT_TOKEN  *TransportToken,
+  IN  MANAGEABILITY_TRANSFER_TOKEN   *TransferToken
+  )
+{
+  EFI_STATUS                                 Status;
+  MANAGEABILITY_TRANSPORT_ADDITIONAL_STATUS  AdditionalStatus;
+
+  if ((TransportToken == NULL) || (TransferToken == NULL)) {
+    DEBUG ((DEBUG_ERROR, "%a: Invalid transport token or transfer token.\n", __func__));
+    return;
+  }
+
+  Status = SsifTransportSendCommand (
+             TransferToken->TransmitHeader,
+             TransferToken->TransmitHeaderSize,
+             TransferToken->TransmitTrailer,
+             TransferToken->TransmitTrailerSize,
+             TransferToken->TransmitPackage.TransmitPayload,
+             TransferToken->TransmitPackage.TransmitSizeInByte,
+             TransferToken->ReceivePackage.ReceiveBuffer,
+             &TransferToken->ReceivePackage.ReceiveSizeInByte,
+             &AdditionalStatus
+             );
+
+  TransferToken->TransferStatus             = Status;
+  TransferToken->TransportAdditionalStatus |= AdditionalStatus;

```

Comment:
Would the value in TransferToken->TransportAdditionalStatus be invalid in the subsequent transfer? Use TransferToken->TransportAdditionalStatus == AdditionalStatus is much safer in case we define some more bit in AdditionalStatus?